skills/gh-address-comments/scripts/fetch_comments.py

```
@@ -117,6 +122,31 @@ def _ensure_gh_authenticated() -> None:
+        graphql = data.get("resources", {}).get("graphql", {})
+        remaining = graphql.get("remaining", None)
+        limit = graphql.get("limit", 5000)
+        reset_at = graphql.get("reset", None)
+        if remaining is None:
+            return
+        pct = remaining / limit * 100
+        if pct < 20:
+            import datetime
+            reset_str = ""
+            if reset_at:
+                reset_dt = datetime.datetime.fromtimestamp(reset_at)
```

Ruff (0.15.10)

[warning] `datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)